fesh/fesh

```diff
@@ -126,6 +162,15 @@ fi
 
 sched=$1
 
+if [[ ! -z "${2:-}" ]]; then
+    FESH_DATA_CENTER=$2
+fi
```

Comment:
Do you want to remove this now you have the `-D` flag?

Reply:
Oops. Thanks.